libs/sdk-backend-tiger/src/backend/organization/index.ts

```diff
@@ -46,8 +47,9 @@ export class TigerOrganizations implements IOrganizations {
 
     public async getCurrentOrganization(): Promise<IOrganization> {
         // TODO: replace with direct call of TigerClient (once methods are generated from OpenAPI)
-        const organizationData = (await this.authCall((sdk) => sdk.axios.get("/api/entities/organization")))
-            .data;
+        const organizationData = (
+            await this.authCall((client) => client.axios.get("/api/entities/organization"))
```

Comment:
Now it's generated on tiger client (we can remove TODO)
`client.organizationObjects.getAllEntitiesOrganizations(...);`

Reply:
I propose to do it as part of followup after your PR. the org utilities getAllPagesOf will be needed here anyway. 

what do you think?